### PR TITLE
722 : Change fill color for rect elements on landing page

### PR DESCRIPTION
**Pull request by DevWig**
## Description
Change fill color for rect elements representing code lines in the sample challenge cards on the landing page to change with the dark and light themes using TailwindCSS

## Related Issue
Issue #722 

## Motivation and Context
The color of the rect elements in the sample challenge cards on the landing page did not update when the theme was changed from light to dark mode
#722 Challenge Card Code Lines Colors

## How Has This Been Tested?
Visual verification with screenshots

## Screenshots/Video (if applicable):
Screenshot 2023-10-01 at 10 45 13 PM
Screenshot 2023-10-01 at 10 45 47 PM

### CODE REVIEW

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 1]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#2B2B32"
+            className="fill-neutral-400 dark:fill-neutral-600"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:79]:
```suggestion
            className="fill-[#ABABb2] dark:fill-[#2B2B32]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 2]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="0"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#544048"
+            className="fill-blue-500 dark:fill-blue-500"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:85]:
```suggestion
            className="fill-[#A48088] dark:fill-[#544048]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 3]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="19"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#544048"
+            className="fill-fuchsia-500 dark:fill-fuchsia-700"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:91]:
```suggestion
            className="fill-[#A48088] dark:fill-[#544048]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 4]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="38"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#404F54"
+            className="fill-fuchsia-500 dark:fill-fuchsia-700"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:97]:
```suggestion
            className="fill-[#809F94] dark:fill-[#404F54]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 5]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             animate={{ width: getRandomWidth() }}
-            fill="#2B2B32"
+            className="fill-neutral-400 dark:fill-neutral-600"
             y="90"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#2B2B32"
+            className="fill-neutral-400 dark:fill-neutral-600"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:109]:
```suggestion
            className="fill-[#ABABb2] dark:fill-[#2B2B32]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 6]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             animate={{ width: getRandomWidth() }}
-            fill="#404F54"
+            className="fill-fuchsia-500 dark:fill-fuchsia-700"
             y="57"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#2B2B32"
+            className="fill-neutral-400 dark:fill-neutral-600"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:103]:
```suggestion
            className="fill-[#ABABb2] dark:fill-[#2B2B32]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 7]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="109"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#544048"
+            className="fill-blue-500 dark:fill-blue-500"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:115]:
```suggestion
            className="fill-[#A48088] dark:fill-[#544048]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 8]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             transition={{ delay: 0.3, duration: 0.5 }}
             y="128"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#544048"
+            className="fill-blue-500 dark:fill-blue-500"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:122]:
```suggestion
            className="fill-[#A48088] dark:fill-[#544048]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 9]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="147"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#404F54"
+            className="fill-fuchsia-500 dark:fill-fuchsia-700"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:128]:
```suggestion
            className="fill-[#809F94] dark:fill-[#404F54]"
```

File: apps/web/src/components/landing/hero-challenge-card.tsx [hunk 10]
```diff
@@ -76,62 +76,62 @@ export function HeroChallengeCard({
...
             y="166"
           />
           <motion.rect
             {...codeLineProps}
             animate={{ width: getRandomWidth() }}
-            fill="#404F54"
+            className="fill-fuchsia-500 dark:fill-fuchsia-700"
```
Comment (PickleNik) [inline apps/web/src/components/landing/hero-challenge-card.tsx:134]:
```suggestion
            className="fill-[#809F94] dark:fill-[#404F54]"
```

**Review by PickleNik — CHANGES_REQUESTED** [review]
great catch on code lines lacking a light mode, but I think these colors  would look better
Screenshot 2023-10-02 at 12 34 26 AM
Screenshot 2023-10-02 at 12 34 42 AM

Edit: actually the no-color lines on dark mode are too dim, keep the neutral-600 for those I think

**Comment by DevWig** [comment]
Requested changes implemented, including the edit to go to a light shade on the dark mode for the no-color lines. I did go with neutral-700 though because neutral-600 looked a bit too light next to the red tinted lines.

**Review by QuiiBz — APPROVED** [review]
Looks good! There is a conflict though, you might want to rebase on main.

**Review by PickleNik — APPROVED** [review]
lgtm

**Pull request merged**